controllers/specialresource_controller.go

```diff
@@ -77,6 +77,7 @@ type SpecialResourceReconciler struct {
 	Storage     storage.Storage
 	KernelData  kernel.KernelData
 	ProxyAPI    proxy.ProxyAPI
+	KubeAPI     clients.ClientsInterface
```

Comment:
How about calling that variable `kubeClient` to be consistent with what we did in some packages?

Reply:
fixed